packages/dom/src/Data/DataSourceItems/MySqlDataSourceItem.ts

```diff
@@ -1,13 +1,16 @@
 import { DataSource } from "../DataSource";
-import { MySqlDataSource } from "../DataSources";
+import { MySqlDataSource, MicrosoftSqlServerDataSource } from "../DataSources";
 import { IProcessDataOnServer } from "../Interfaces/IProcessDataOnServer";
 import { ProcedureDataSourceItem } from "../ProcedureDataSourceItem";
 
 
 export class MySqlDataSourceItem extends ProcedureDataSourceItem implements IProcessDataOnServer {
-    constructor(title: string, dataSource: DataSource) {
+    constructor(title: string, dataSource: DataSource, table?: string) {
```

Comment:
Do not change the ctor of these data sources. The web works differently than WPF. Once we have the Data source working, we can create a ctor API that makes sense. We may not even make use of these objects directly but rather use a factory to create these obejcts.